generator/integration_tests/golden/internal/golden_thing_admin_rest_logging_decorator.h

```diff
@@ -0,0 +1,122 @@
+
+  Status DropDatabase(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::DropDatabaseRequest const& request) override;
+
+  StatusOr<google::test::admin::database::v1::GetDatabaseDdlResponse> GetDatabaseDdl(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::GetDatabaseDdlRequest const& request) override;
+
+  StatusOr<google::iam::v1::Policy> SetIamPolicy(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::iam::v1::SetIamPolicyRequest const& request) override;
+
+  StatusOr<google::iam::v1::Policy> GetIamPolicy(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::iam::v1::GetIamPolicyRequest const& request) override;
+
+  StatusOr<google::iam::v1::TestIamPermissionsResponse> TestIamPermissions(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::iam::v1::TestIamPermissionsRequest const& request) override;
+
+  StatusOr<google::longrunning::Operation> CreateBackup(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::CreateBackupRequest const& request) override;
+
+  StatusOr<google::test::admin::database::v1::Backup> GetBackup(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::GetBackupRequest const& request) override;
+
+  StatusOr<google::test::admin::database::v1::Backup> UpdateBackup(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::UpdateBackupRequest const& request) override;
+
+  Status DeleteBackup(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::DeleteBackupRequest const& request) override;
+
+  StatusOr<google::test::admin::database::v1::ListBackupsResponse> ListBackups(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::ListBackupsRequest const& request) override;
+
+  StatusOr<google::longrunning::Operation> RestoreDatabase(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::RestoreDatabaseRequest const& request) override;
+
+  StatusOr<google::test::admin::database::v1::ListDatabaseOperationsResponse> ListDatabaseOperations(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::ListDatabaseOperationsRequest const& request) override;
+
+  StatusOr<google::test::admin::database::v1::ListBackupOperationsResponse> ListBackupOperations(
+      google::cloud::rest_internal::RestContext& rest_context,
+      google::test::admin::database::v1::ListBackupOperationsRequest const& request) override;
+
+ private:
+  std::shared_ptr<GoldenThingAdminRestStub> child_;
+  TracingOptions tracing_options_;
+  std::set<std::string> components_;
+};  // GoldenThingAdminRestLogging
```

Comment:
This closing comment seems useless.  And we only generated it for the `*Logging` decorators, can we remove it?